webview_panels/src/modules/queryPanel/components/perspective/perspective.module.scss

```diff
@@ -1,6 +1,7 @@
 .altimatePerspectiveViewer {
   height: 100%;
   font-size: 1.25rem !important;
+  z-index: 20;
```

Comment:
why is this needed?

Reply:
Because perspective viewer's configure button shows up on the Drawer.